js/impress.js

```diff
@@ -716,12 +716,19 @@
                     case 40: // down
                              api.next();
                              break;
+                    case 36: // character o
+                    case 79: // home 
+                            var overview = document.getElementById("overview");
+                            if (overview) {
+                                api.goto(overview);
+                            }
+                            break; 
```

Comment:
the above could just be `api.goto("overview")` which will do nothing if such an element is not found.